### ISSUE TITLE
[bug]: Deleting from the gallery intermittently fails

### ISSUE THREAD

**Issue by Fortyseven** [invoke-ai/InvokeAI#1295]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Linux

### GPU

cuda

### VRAM

24

### What happened?

- Start server
- Generate image
- Immediately delete image from gallery (trash icon or delete key)
- 'Disconnected' in console
  - (Rinse, repeat for as many you like.)
- When client immediately reconnects 'Deleted' toast pops up, but image(s) remain in gallery list
- Reload client browser tab
- Image(s) have been deleted

### Client console:
```
2022-10-29T18:52:56-0400: Image generation requested: {"prompt":"test prompt","iterations":1,"steps":6,"cfg_scale":8,"threshold":0,"perlin":0,"height":512,"width":512,"sampler_name":"k_dpm_2_a","seed":1497525810,"progress_images":true,"seamless":false,"hires_fix":false,"variation_amount":0}
2022-10-29T18:52:58-0400: Image generated: outputs/000194.8863c182.1497525810.png
2022-10-29T18:53:01-0400: Disconnected from server
2022-10-29T18:53:02-0400: Loaded 0 images
2022-10-29T18:53:02-0400: Loaded 0 images
```
### Server side console:
```
* Initializing, be patient...
>> GFPGAN Initialized
>> CodeFormer Initialized
>> ESRGAN Initialized
>> Using device_type cuda
>> Loading stable-diffusion-1.4 from models/ldm/stable-diffusion-v1/model.ckpt
   | LatentDiffusion: Running in eps-prediction mode
   | DiffusionWrapper has 859.52 M params.
   | Making attention of type 'vanilla' with 512 in_channels
   | Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
   | Making attention of type 'vanilla' with 512 in_channels
   | Using faster float16 precision
>> Model loaded in 8.45s
>> Max VRAM used to load the model: 2.17G 
>> Current VRAM usage:2.17G
>> Setting Sampler to k_lms

* --web was specified, starting web server...
>> Started Invoke AI Web Server!
>> Default host address now 127.0.0.1 (localhost). Use --host 0.0.0.0 to bind any address.
>> Point your browser at http://127.0.0.1:9090
>> Image generation requested: {'prompt': 'test prompt', 'iterations': 1, 'steps': 6, 'cfg_scale': 8, 'threshold': 0, 'perlin': 0, 'height': 512, 'width': 512, 'sampler_name': 'k_dpm_2_a', 'seed': 1497525810, 'progress_images': True, 'seamless': False, 'hires_fix': False, 'variation_amount': 0}
ESRGAN parameters: False
Facetool parameters: False
>> Setting Sampler to k_dpm_2_a
>> Parsed prompt to FlattenedPrompt:[Fragment:'test prompt'@1.0]
Generating:   0%|                                                                     | 0/1 [00:00<?, ?it/s]>> Ksampler using karras noise schedule (steps <= 29)
>> Sampling with k_dpm_2_ancestral starting at step 0 of 6 (6 new sampling steps)
100%|█████████████████████████████████████████████████████████████████████████| 6/6 [00:01<00:00,  3.76it/s]
>> Image generated: "/home/fortyseven/opt/ai/invoke-ai/outputs/img-samples/000194.8863c182.1497525810.png"s]
Generating: 100%|█████████████████████████████████████████████████████████████| 1/1 [00:01<00:00,  1.83s/it]
>> Usage stats:
>>   1 image(s) generated in 2.26s
>>   Max VRAM used for this generation: 3.40G. Current VRAM utilization: 2.25G
>>   Max VRAM used since script start:  3.40G
>> Delete requested "outputs/000194.8863c182.1497525810.png"
/home/fortyseven/opt/ai/invoke-ai/outputs/img-samples/000194.8863c182.1497525810.png
```

**Comment by hipsterusername** [comment]
@psychedelicious @blessedcoolant - FYI, also experienced the same last night.

**Comment by psychedelicious** [comment]
Fixed in #1301.
